### PR TITLE
Add Fleet/Agent 8.7.0 release notes

### PR DESCRIPTION
**Pull request by kilfoyle**
This adds the 8.7.0 Release Notes. See [Preview page](https://observability-docs_2788.docs-preview.app.elstc.co/guide/en/fleet/master/release-notes-8.7.0.html).   ← This preview site seems not to be updating.  `¯\_(ツ)_/¯`

**Notes**:

 - Elastic Agent notes are ready. ~Elastic Agent notes are still TBD. @cmacknz will ensure that the [changelog tool](https://github.com/elastic/elastic-agent-changelog-tool) is run before release day and I'll take it from there.~
 - For Fleet Server, I used this [fragments list](https://github.com/elastic/fleet-server/tree/4854b153a3d77e898993193ad6af1b52176850d5/changelog/fragments) which contained just one fragment. If anything more should be added, please let me know.
 - The `kubernetes-conditions-autodiscover.asciidoc` file is included because it contained a broken link to the 8.6 release notes, which is fixed in this PR.

Closes: https://github.com/elastic/ingest-docs/issues/60

### CODE REVIEW

**Review by kpollich — APPROVED** [review]
Fleet notes LGTM 🚀

File: docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+impact to your application.
+
+[discrete]
+[[breaking-147616]]
+.Remove the current_upgrades endpoint
+[%collapsible]
+====
+*Details* +
+The `api/fleet/current_upgrades` endpoint has been removed. For more information, refer to {kibana-pull}147616[#147616].
+*Impact* +
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc:46]:
```suggestion

*Impact* +
```

Comment (dedemorton) [reply docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc:46]: You need a line break here. 
image

File: docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+====
+
+[discrete]
+[[breaking-147199]]
+.Remove the preconfiguration API route
+[%collapsible]
+====
+*Details* +
+The `/api/fleet/setup/preconfiguration` API, which was released as generally available by error, has been removed. For more information, refer to {kibana-pull}147199[#147199].
+*Impact* +
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc:57]:
```suggestion

*Impact* +
```

File: docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+{fleet}::
+* Add ability to show FQDN of agents {kibana-pull}150239[#150239]
+* Add `getStatusSummary` query parameter to `GET /api/fleet/agents` API {kibana-pull}149963[#149963]
+* Enable diagnostics feature flag and change query for files to use upload_id {kibana-pull}149575[#149575]
+* Add experimental toggles for doc-value-only {kibana-pull}149131[#149131]
+* Display agent metrics, CPU and memory in the agent list table and agent details page {kibana-pull}149119[#149119]
+* Implement subcategories in integrations UI {kibana-pull}148894[#148894]
+* Add rollout period to upgrade action {kibana-pull}148240[#148240]
+* Add per-policy inactivity timeout + use runtime fields for agent status {kibana-pull}147552[#147552]
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc:87]: Can you replace + with and here?

File: docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+* Show dataset combo box for input packages {kibana-pull}147015[#147015]
+* Add UI controls to setting/outputs to configure new shipper {kibana-pull}145755[#145755]
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc:89]: I don't know what this means without following the pr link

**Review by dedemorton — COMMENTED** [review]
Don't have time for anything more than a cursory review, but this is what I noticed.

File: docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc (hunk 3 above)
Comment (kilfoyle) [reply docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc]: Addressed in [b3a983d](https://github.com/elastic/observability-docs/pull/2788/commits/b3a983d65af3b2d0495f85fb06f787fd1bca0ec0)

File: docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+* add info
+
+[discrete]
+[[new-features-8.7.0]]
+=== New features
+
+The 8.7.0 release adds the following new and notable features.
+
+{fleet}::
+* Add ability to show FQDN of agents {kibana-pull}150239[#150239]
```
Comment (cmacknz) [inline docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc:82]:
```suggestion
```

This won't work properly until 8.7.1. We shouldn't announce it in 8.7.0. The changes to the Elastic Agent aren't fully merged, it was always planned that we would enable this in 8.7.1 to give us more testing time.

**Comment by cmacknz** [comment]
I've generated the changelog summary for the 8.7.0 Elastic Agent in https://github.com/elastic/elastic-agent/pull/2395.

**Comment by kilfoyle** [comment]
Thanks a lot @cmacknz! 
I've ported in the [8.7.0 changelog entries](https://github.com/elastic/elastic-agent/pull/2395).

**Comment by cmacknz** [comment]
The agent release notes look good, thanks!

We the following two known issues to carry forward into the 8.7.0 release notes:

https://github.com/elastic/observability-docs/blob/b136b51f5348e64325f9493c3b3427179e37b158/docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc?plain=1#L37-L76

**Review by cmacknz — APPROVED** [review]
Thanks!

**Pull request merged**